taco-tuesday-rest-api/src/main/java/com/muskopf/tacotuesday/domain/FullOrder.java

```diff
@@ -1,20 +1,23 @@
 package com.muskopf.tacotuesday.domain;
 
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+
 import javax.persistence.*;
 import java.util.Set;
 
 @Entity
+@JsonIdentityInfo(
+        generator = ObjectIdGenerators.PropertyGenerator.class,
+        property = "id"
+)
 public class FullOrder extends Order {
-    @Column(name = "fk_full_order_id")
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    Integer fullOrderId;
-
-    @OneToMany(mappedBy = "fullOrder", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    @OneToMany(mappedBy = "fullOrder", fetch = FetchType.EAGER, cascade = CascadeType.MERGE)
+    @JsonManagedReference
```

Comment:
This tells Jackson that it's the _owning_ side of the bidirectional relationship